### PR TITLE
docs(nx-dev): revamp the Nx Enterprise page on nx.dev

### PR DESCRIPTION
**Pull request by bcabanes**
This update introduces a revamp to the Nx Enterprise page. The code changes involved the addition of new image files, amending several components for improved UI/UX. This refactor also includes an alteration to the Call-to-Action section and an introduction of a new Carousel component for better navigation. These adjustments aim to provide an improved user experience and enhanced readability.

### CODE REVIEW

File: nx-dev/ui-enterprise/src/lib/make-your-ci-fast.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                  <span className="inline-flex items-center rounded-md bg-slate-400/10 px-2 py-1 text-xs font-medium text-slate-400 ring-1 ring-inset ring-slate-400/20">
+                    Flaky task retries
+                  </span>
+                </div>
+                <h3 className="text-lg font-medium tracking-tight text-slate-950 dark:text-white">
+                  One less thing to debug
+                </h3>
+                <p className="mt-2 max-w-lg text-sm/6">
+                  Automatically detects and re-runs flaky tasks, enhancing the
+                  reliability of your CI processes and minimize time spent
```
Comment (isaacplmann) [inline nx-dev/ui-enterprise/src/lib/make-your-ci-fast.tsx:57]:
```suggestion
                  reliability of your CI processes and minimizing time spent
```

File: nx-dev/ui-enterprise/src/lib/make-your-ci-fast.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                    Atomizer
+                  </span>
+                </div>
+                <h3 className="text-lg font-medium tracking-tight text-slate-950 dark:text-white">
+                  Break tasks down, to speed tests up
+                </h3>
+                <p className="mt-2 max-w-lg text-sm/6">
+                  <span className="font-semibold">Atomizer</span> automatically
+                  splits large e2e tests into smaller, atomized, tasks, enabling
+                  lightening fast testing. Parallelize your tests to reduce
```
Comment (isaacplmann) [inline nx-dev/ui-enterprise/src/lib/make-your-ci-fast.tsx:82]:
```suggestion
                  lightning fast testing. Parallelize your tests to reduce
```

File: nx-dev/ui-enterprise/src/lib/scale-your-organization.tsx [hunk 3]
```diff
@@ -0,0 +1,317 @@
...
+                  <div className="size-1.5 rounded-full bg-slate-100 ring-1 ring-slate-300 dark:bg-slate-800 dark:ring-slate-600" />
+                </div>
+                <div className="flex flex-col pb-10">
+                  <dt className="text-base font-semibold leading-7 text-black dark:text-white">
+                    <div className="mb-6 flex">
+                      <div className="rounded-lg bg-blue-500 px-2 py-0.5 font-semibold text-white dark:bg-white dark:text-black">
+                        Visibility
+                      </div>
+                    </div>
+                    See dependencies across team and repos
```
Comment (isaacplmann) [inline nx-dev/ui-enterprise/src/lib/scale-your-organization.tsx:105]:
```suggestion
                    See dependencies across teams and repos
```

File: nx-dev/ui-enterprise/src/lib/scale-your-organization.tsx [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+                        Included in{' '}
+                        <Link
+                          href="/powerpack"
+                          title="Nx Powerpack"
+                          prefetch={false}
+                          className="underline"
+                        >
+                          Nx Powerpack
+                        </Link>
+                        , available complimentary to all Enterprise customers.
```
Comment (isaacplmann) [inline nx-dev/ui-enterprise/src/lib/scale-your-organization.tsx:171]:
```suggestion
                        , and available complimentary to all Enterprise customers.
```

File: nx-dev/ui-enterprise/src/lib/scale-your-organization.tsx [hunk 5]
```diff
@@ -0,0 +1,317 @@
...
+                      <i>
+                        Included in{' '}
+                        <Link
+                          href="/powerpack"
+                          title="Nx Powerpack"
+                          className="underline"
+                        >
+                          Nx Powerpack
+                        </Link>
+                        , available complimentary to all Enterprise customers.
```
Comment (isaacplmann) [inline nx-dev/ui-enterprise/src/lib/scale-your-organization.tsx:254]:
```suggestion
                        , and available complimentary to all Enterprise customers.
```

File: nx-dev/ui-enterprise/src/lib/scale-your-organization.tsx [hunk 6]
```diff
@@ -0,0 +1,317 @@
...
+                    <div className="mb-6 flex">
+                      <div className="rounded-lg bg-blue-500 px-2 py-0.5 font-semibold text-white dark:bg-white dark:text-black">
+                        Ownership
+                      </div>
+                    </div>
+                    Clear, expressive ownership
+                  </dt>
+                  <dd className="mt-1 flex flex-auto flex-col text-base leading-7">
+                    <p className="flex-auto">
+                      Unlike the code ownership tools from VCS provides, Nx
```
Comment (isaacplmann) [inline nx-dev/ui-enterprise/src/lib/scale-your-organization.tsx:237]:
```suggestion
                      Unlike the code ownership tools from VCS providers, Nx
```

File: nx-dev/ui-enterprise/src/lib/scale-your-organization.tsx [hunk 7]
```diff
@@ -0,0 +1,317 @@
...
+        <rect
+          width="100%"
+          height="100%"
+          strokeWidth={0}
+          fill="url(#1d4240dd-898f-445f-932d-e2872fd12de3)"
+        />
+      </svg>
+      <div className="mx-auto max-w-5xl text-center">
+        <SectionHeading as="h2" variant="title" id="scale-your-organization">
+          Don’t lose output as your organization grows{' '}
```
Comment (isaacplmann) [inline nx-dev/ui-enterprise/src/lib/scale-your-organization.tsx:307]:
```suggestion
          Don’t lose velocity as your organization grows{' '}
```

**Review by vsavkin — APPROVED** [review]

**Review by isaacplmann — CHANGES_REQUESTED** [review]
A few typos and wording updates.

And the Flaky Task Retries image has a white line on the left when the browser window is 900px wide.

Screenshot 2024-12-05 at 4 32 21 PM

**Review by isaacplmann — APPROVED** [review]

**Pull request merged**